Client/Frontend/Widgets/Theme.swift

```diff
@@ -1,13 +1,54 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
 import Foundation
 
 protocol Themeable {
+    
     func applyTheme(_ theme: Theme)
+    
 }
 
 enum Theme: String {
-    case Private
-    case Normal
+    
+    /// Regular browsing.
+    case regular
+
+    /// Private browsing.
+    case `private`
+    
+    /// Textual representation suitable for debugging.
+    var debugDescription: String {
+        switch self {
+        case .regular:
+            return "Regular theme"
+        case .private:
+            return "Private theme"
+        }
+    }
+
+    /// Returns whether the theme is private or not.
+    var isPrivate: Bool {
+        switch self {
```

Comment:
Are you switching because it's possible a future tab type would also use the `private` theme? If thats the case, wouldn't it be better to drop the "private" theme entirely for just "dark"?

Reply:
Yeah, I think that makes more sense.

Reply:
👍

Reply:
Change to dark/light themes brings up too many questions for this PR, I will create a new ticket for themes